fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -983,7 +983,7 @@ private void handleDropStatsStmt() {
         DropStatsStmt dropStatsStmt = (DropStatsStmt) parsedStmt;
         Table table = MetaUtils.getTable(context, dropStatsStmt.getTableName());
         if (dropStatsStmt.isExternal()) {
-            GlobalStateMgr.getCurrentAnalyzeMgr().dropExternalStats(table.getUUID());
+            GlobalStateMgr.getCurrentAnalyzeMgr().dropExternalStats(table.getUUID(), dropStatsStmt.getTableName().getCatalog());
```

Comment:
You can define an interface in the ConnectorMetadata that are similar to invalidateXXX.

Reply:
as what I say in former, metadata is query level, invalidate is catalog level, it's not suitable. and drop ExternalStats contains invalidate cache, remove stats status, remote stats data in be.